src/WebViewTypes.ts

```diff
@@ -542,6 +626,33 @@ export interface AndroidWebViewProps extends WebViewSharedProps {
    * Sets ability to open fullscreen videos on Android devices.
   */
   allowsFullscreenVideo?: boolean;
+
+  /**
+   * A Base URL that is intercepted and sent through onUrlSchemeRequest. Android has a few issues
+   * with custom schemes, using a baseInterceptURL instead.
+   */
+  baseInterceptUrl?: string;
```

Comment:
How different would the value passed here be from iOS urlScheme props (trying to see if we can unite thoses)

Reply:
See comment below about the differences between Android and iOS.